### ISSUE TITLE
Feature Request: Switch Output and Input device with one hot key

### ISSUE THREAD

**Issue by ssshake** [xenolightning/AudioSwitcher_v1#2230]
I have two set ups that I want to toggle between with one button.

In Mode "A" I want audio output to my monitor over display port and I want my microphone set to my Yeti desktop mic.

In Mode "B" I want my audio output to my steel series wireless headphone, and I also want my microphone set to the headphone.

Currently I have 4 hot keys, two for output and two for input. 

It would be awesome to have only 2 hot keys. One for desktop speak/mic mode and one for headphone mode.

Even better would be a toggle. Desktop mode or headphone mode.

My main reason is I really don't like wearing headphone so most of the time I want desktop mode. But when gaming or when making food in the kitchen when I'm on a conference call I would like to quickly switch to headphone mode.

**Comment by erovneiko** [comment]
Hello,

Good idea! I also would be appreciated for such feature. 

Also would be nice to have just one hot key to cycle between listed devices (input/output pairs) with no separate hot keys for each.

And hotkey for media pause function.

**Comment by mblade** [comment]
Yes, please do this! I specifically found this software to switch between profiles of devices. I go from wireless headphones to VR a lot, and its a pain to find the menu. Windows buries it deeper and deeper. Please push an update for this feature! Until then I'll have to uninstall and keep on looking.
